Packs/CommonPlaybooks/ReleaseNotes/2_3_52.md

```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Account Enrichment - Generic v2.1
+
+- Added filter to check if inputs.Domain Is not empty on task `IAM Get User` .
```

Comment:
I rather say we improved the filters and the transformers on the "IAM Get User" task (cause we didn't change the logic itself).

Reply:
Done